packages/app/id-utils/package.json

```diff
@@ -1,53 +1,48 @@
 {
-	"name": "@lokalise/id-utils",
-	"version": "2.1.0",
-	"type": "module",
-	"files": [
-		"dist",
-		"README.md",
-		"LICENSE.md"
-	],
-	"license": "Apache-2.0",
-	"main": "./dist/index.cjs",
-	"module": "./dist/index.js",
-	"types": "./dist/index.d.ts",
-	"homepage": "https://github.com/lokalise/shared-ts-libs",
-	"repository": {
-		"type": "git",
-		"url": "git://github.com/lokalise/shared-ts-libs.git"
-	},
-	"exports": {
-		".": {
-			"types": "./dist/index.d.ts",
-			"require": "./dist/index.cjs",
-			"import": "./dist/index.js"
-		}
-	},
-	"scripts": {
-		"build": "rimraf dist && vite build",
-		"dev": "vite watch",
-		"clean": "rimraf dist .eslintcache",
-		"lint": "eslint --cache --max-warnings=0  && prettier --check --log-level warn src \"**/*.{json,md}\" && tsc --noEmit",
-		"lint:fix": "eslint  --fix && prettier --write src \"**/*.{json,md}\"",
-		"test:ci": "vitest run --coverage",
-		"prepublishOnly": "npm run build",
-		"package-version": "echo $npm_package_version"
-	},
-	"dependencies": {
-		"ulid-uuid-converter": "^1.0.4",
-		"ulidx": "^2.3.0",
-		"uuidv7": "^1.0.1"
-	},
-	"devDependencies": {
-		"@lokalise/eslint-config": "latest",
-		"@lokalise/package-vite-config": "latest",
-		"@lokalise/prettier-config": "latest",
-		"@vitest/coverage-v8": "^2.0.4",
-		"prettier": "^3.3.3",
-		"rimraf": "^5.0.7",
-		"typescript": "5.5.4",
-		"vite": "5.3.5",
-		"vitest": "^2.0.4"
-	},
-	"prettier": "@lokalise/prettier-config"
+  "name": "@lokalise/id-utils",
+  "version": "2.1.0",
+  "type": "module",
+  "files": ["dist", "README.md", "LICENSE.md"],
+  "license": "Apache-2.0",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "homepage": "https://github.com/lokalise/shared-ts-libs",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/lokalise/shared-ts-libs.git"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js"
+    }
+  },
+  "scripts": {
+    "build": "rimraf dist && vite build",
+    "dev": "vite watch",
+    "clean": "rimraf dist .eslintcache",
+    "lint": "biome check . && tsc --project tsconfig.lint.json --noEmit",
+    "lint:fix": "biome check --write",
+    "test:ci": "vitest run --coverage",
+    "prepublishOnly": "npm run build",
+    "package-version": "echo $npm_package_version",
+    "postversion": "biome check --write package.json"
+  },
+  "dependencies": {
+    "ulid-uuid-converter": "^1.0.4",
+    "ulidx": "^2.3.0",
+    "uuidv7": "^1.0.1"
+  },
+  "devDependencies": {
+    "@biomejs/biome": "^1.8.3",
+    "@lokalise/biome-config": "^1.3.0",
+    "@lokalise/package-vite-config": "latest",
+    "@vitest/coverage-v8": "^2.0.4",
+    "rimraf": "^5.0.7",
```

Comment:
let's bump this to 6.x now that we are at it

Reply:
Done, thank youuu 🙏